.github/workflows/release.yml

```
@@ -46,10 +51,37 @@ jobs:
           echo "::set-output name=name::${GITHUB_REF#refs/*/}"
           echo "::set-output name=version::${GITHUB_REF##*/}"
         id: extract_branch
+      - name: Check npm version preid
+        run: |
+          HAS_PREID=$(node -p "'${{ steps.extract_branch.outputs.version }}'.startsWith('prerelease-') ? 1 : 0")
+          echo "::set-output name=has_preid::${HAS_PREID}"
+          if [ $HAS_PREID -eq 1 ]; then
```

[nits] `$HAS_PREID` には値が入るので多分気にしなくていいんですが

```suggestion
          if [ "$HAS_PREID" -eq 1 ]; then
```

`"` で囲ってあげた方が構文エラーが絶対にでないので嬉しそうかなと思いました。

---

https://github.com/openameba/cwv-logger/pull/8/commits/c82e1e43ea90f3dac6328c95039011d6ce20b3cf
こちらで修正しました！